src/classes/ERR_Notifier.cls

```diff
@@ -68,13 +70,13 @@ public class ERR_Notifier {
             //We will send email
             } else {
                 List<String> sendList = new List<String>();
-                if (errorNotifRecipient instanceof id && errorNotifRecipient.startsWith(NotificationOptions.user)) {
+                if (errorNotifRecipient instanceof id && errorNotifRecipient.startsWith(NotificationOptions.user) && context != CONTEXT_USER) {
```

Comment:
why the added contact != CONTEXT_USER?  not sure I understand the scenario.  thanks.

Reply:
The context check was used to support the async handling of the Error Notification Settings. In the earlier version of UserService (prior to check in), it was possible that the settings were still set to the InActive/Frozen User. However, this check is currently no longer necessary so I've removed it. Thanks!